gtsam/hybrid/HybridBayesNet.h

```diff
@@ -118,6 +118,29 @@ class GTSAM_EXPORT HybridBayesNet : public BayesNet<HybridConditional> {
     return evaluate(values);
   }
 
+  /**
+   * @brief Assemble a DecisionTree of (GaussianBayesNet, double) leaves for
+   * each discrete assignment.
+   * The included double value is used to make
+   * constructing the model selection term cleaner and more efficient.
+   *
+   * @return GaussianBayesNetValTree
+   */
+  GaussianBayesNetValTree assembleTree() const;
+
+  /*
+    Perform the integration of L(X;M,Z)P(X|M)
+    which is the model selection term.
+
+    By Bayes' rule, P(X|M,Z) ∝ L(X;M,Z)P(X|M),
+    hence L(X;M,Z)P(X|M) is the unnormalized probabilty of
+    the joint Gaussian distribution.
+
+    This can be computed by multiplying all the exponentiated errors
+    of each of the conditionals.
+  */
+  AlgebraicDecisionTree<Key> model_selection() const;
```

Comment:
naming convention: `modelSelection`. Although, it's a bit of a weird name, and the comment does not help much. What is the return value?

Reply:
Done! Hopefully the improved docstring is better.